### PR TITLE
[no-relnote] Fix release tooling

### PR DESCRIPTION
**Pull request by elezar**

### CODE REVIEW

File: hack/prepare-artifacts.sh [hunk 1]
```diff
@@ -47,7 +47,44 @@ PACKAGE_ROOT=release-${VERSION}-${REPO}
...
+  elif is_command openssl; then
+    hash=$(openssl -dst openssl dgst -sha256 "$TARGET") || return 1
+    echo "$hash" | cut -d ' ' -f a
+  else
+    log_err "hash_sha256 unable to find command to compute sha-256 hash"
+    return 1
+  fi
+)
+
+
```
Comment (ArangoGutierrez) [inline hack/prepare-artifacts.sh:79]: nit trailing new lines

File: hack/create-release.sh [hunk 2]
```diff
@@ -31,7 +31,7 @@ REPOSITORY=NVIDIA/nvidia-container-toolkit
 echo "Creating draft release"
 ./hack/generate-changelog.sh --version ${VERSION} | \
     grep -v "### Version v" | \
```
Comment (ArangoGutierrez) [inline hack/create-release.sh:33]: generate-changelog doesn't have a string `### Version v`,  this match won't happen

Comment (elezar) [reply hack/create-release.sh:33]: You're right. It's an inverted match anyway, so we'll skip something that's not there.

**Review by ArangoGutierrez — APPROVED** [review]
tested all the bash bits, can't test the `gh` bits since it doesn't provide a dry run option. But PR looks good

**Pull request merged**